packages/@o3r/eslint-config-otter/schematics/ng-update/v11.4/update-configs/update-configs.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { Tree } from '@angular-devkit/schematics';
+import { updateEslintRecommended } from './update-configs';
+
+describe('updateEslintRecommended', () => {
+  it('should update configs', async () => {
+    const initialTree = Tree.empty();
+    initialTree.create('random.file', 'a file containing json-recommended');
+    initialTree.create('/test/.eslintrc.json', '{ "extends": ["@o3r/json-recommended", "@other/json-recommended"] }');
+    initialTree.create('/src/.eslintrc.json', '{ "extends": ["plugin:@o3r/json-recommended", "@o3r/template-recommended"] }');
```

Comment:
You don't have test for `eslint.config.{m,c,}js` that match the regexp `/^.?eslint(:?rc)?\..*/`